packages/api-catalog-spa/src/pages/PlaygroundView/PlaygroundView.tsx

```diff
@@ -0,0 +1,11 @@
+import { Playground, store } from 'graphql-playground-react';
+import { Provider } from 'react-redux';
+
```

Comment:
Interface for this components prop is missing.

Reply:
Updated